wcfsetup/install/files/ts/WoltLabSuite/Core/Ui/Message/Quote.ts

```diff
@@ -0,0 +1,553 @@
+import * as Ajax from "../../Ajax";
+import * as Core from "../../Core";
+import * as EventHandler from "../../Event/Handler";
+import * as Language from "../../Language";
+import DomChangeListener from "../../Dom/Change/Listener";
+import DomUtil from "../../Dom/Util";
+import { AjaxCallbackObject, AjaxCallbackSetup } from "../../Ajax/Data";
+
+interface AjaxResponse {
+  actionName: string;
+  returnValues: {
+    count?: number;
+    fullQuoteMessageIDs?: unknown;
+    fullQuoteObjectIDs?: unknown;
+    renderedQuote?: string;
+  };
+}
+
+interface ElementBoundaries {
+  bottom: number;
+  left: number;
+  right: number;
+  top: number;
+}
+
+class UiMessageQuote implements AjaxCallbackObject {
```

Comment:
```suggestion
export class UiMessageQuote implements AjaxCallbackObject {
```
Convention is both the regular export + default export, because both have different benefits.